Cards/CardBase.gd

```diff
@@ -82,3 +89,11 @@ func _apply_buffs(caster: Entity, target: Entity) -> void:
 				party_member.get_buff_component().add_buff(buff, caster)
 		else:
 			target.get_buff_component().add_buff(buff, caster)
+
+
+func _draw_cards():
+	CardManager.card_container.draw_cards(amount_of_cards_to_draw)
+
+
+func _discard_cards():
```

Comment:
maybe this should be called "discard_random_cards" as opposed to a function that would let you choose which cards to discard for example

Reply:
Good call, will change